internal/website/content/howto/blob/_index.md

```diff
@@ -128,16 +138,21 @@ other API that takes in a `*gcp.HTTPClient`.) You can find functions in the
 
 {{< goexample "gocloud.dev/blob/gcsblob.ExampleOpenBucket" >}}
 
-[GCP creds]: https://cloud.google.com/docs/authentication/production
 [`gcsblob.OpenBucket`]: https://godoc.org/gocloud.dev/blob/gcsblob#OpenBucket
 [`gocloud.dev/gcp`]: https://godoc.org/gocloud.dev/gcp
 
 ### S3 {#s3}
 
 S3 URLs in the Go CDK closely resemble the URLs you would see in the AWS CLI.
-You can specify the `region` query parameter to ensure your application connects
-to the correct region, but otherwise `blob.OpenBucket` will use the region found
-in the environment variables or your AWS CLI configuration.
+You should specify the `region` query parameter to ensure your application
```

Comment:
Move this to the end saying like before "you should specify.... , otherwise will use ..."?

Reply:
I weakened this comment because I've found that I've had to specify `region` in the URL or it doesn't work. I think it's because while it will default to using a region from environment variables or you AWS CLI configuration, it's not normal to set either of those things (i.e., if you just `aws login` or whatever, the region is not set). So I think it's better to tell people to just put it in the URL.